plugins/by-name/dap-python/default.nix

```
@@ -0,0 +1,94 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    optionalString
+    types
+    ;
+  inherit (lib.nixvim)
+    defaultNullOpts
+    mkNullOrOption
+    toLuaObject
+    ;
+
+  dapHelpers = import ../dap/dapHelpers.nix { inherit lib; };
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "dap-python";
+  package = "nvim-dap-python";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  extraOptions = {
+    adapterPythonPath = lib.mkOption {
+      default = lib.getExe (pkgs.python3.withPackages (ps: with ps; [ debugpy ]));
+      defaultText = lib.literalExpression ''
+        lib.getExe (pkgs.python3.withPackages (ps: with ps; [ debugpy ]))
+      '';
+      description = "Path to the python interpreter. Path must be absolute or in $PATH and needs to have the debugpy package installed.";
```

Nit: 
```suggestion
      description = "Path to the python interpreter. Path must be absolute or in `$PATH` and needs to have the `debugpy` package installed.";
```

Also: feels like the wording could be improved?

Might be getting long enough to justify using an `'' ''` indented string?